lib/App/perlimports/Document.pm

```diff
@@ -8,7 +8,7 @@ our $VERSION = '0.000009';
 use App::perlimports::Annotations ();
 use App::perlimports::Include     ();
 use File::Basename qw( fileparse );
-use List::Util qw( any uniq );
+use List::Util qw( any uniq);
```

Comment:
Ha! You're trying to defeat my padding. ;)

Reply:
heh, sorry